tests/testthat/test-expressions.R

```diff
@@ -144,6 +144,13 @@ with_mock_crunch({
         print(ds$birthyr * 3 + 5)
         print(3 * (ds$birthyr + 5))
     })
+
+    test_that("rollup functions generate expected requests", {
+        expect_identical(rollupResolution(ds$starttime), "s")
+        expect_PATCH(rollupResolution(ds$starttime) <- "M",
+            'https://app.crunch.io/api/datasets/1/variables/starttime/',
+            '{"view":{"rollup_resolution":"M"}}')
+    })
 })
 
 with_test_authentication({
```

Comment:
I'm not sure if it's worth the added functional test here, but it might be good to add one — just to make sure that the API doesn't change without us realizing. 

If it's well tested in backend-http, and it would be slow here, then maybe we should skip here.